python/pyspark/ml/classification.py

```diff
@@ -44,6 +44,23 @@
 
 
 @inherit_doc
+class JavaClassificationModel(JavaPredictionModel):
+    """
+    (Private) Java Model produced by a ``Classifier``.
+    Classes are indexed {0, 1, ..., numClasses - 1}.
+    To be mixed in with class:`pyspark.ml.JavaModel`
+    """
+
+    @property
+    @since("2.0.0")
```

Comment:
Ah yes good point, this PR has been around - I'll update it here and in the other new method too.